dashboard/README.md

```diff
@@ -1,70 +1,198 @@
+    │   │   │   │   ├── index.less
+    │   │   │   │   ├── LoginForm.jsx
+    │   │   │   │   ├── signupFormData.js
+    │   │   │   │   └── SignupForm.jsx
+    │   │   │   ├── BackgroundComponent
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   ├── BreadCrumbComponent
+    │   │   │   │   └── index.jsx
+    │   │   │   ├── DatePickerComponent
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   ├── HeaderComponent
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   ├── NavbarComponent
+    │   │   │   │   └── index.jsx
+    │   │   │   ├── NavbarDrawerComponent
+    │   │   │   │   └── index.jsx
+    │   │   │   ├── OverviewComponent
+    │   │   │   │   ├── common-component.jsx
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   ├── index.less
+    │   │   │   │   └── NewRunsComponent.jsx
+    │   │   │   ├── PaginationComponent
+    │   │   │   │   └── index.jsx
+    │   │   │   ├── ProfileComponent
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   ├── SidebarComponent
+    │   │   │   │   └── index.jsx
+    │   │   │   ├── TableComponent
+    │   │   │   │   ├── common-components.jsx
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   ├── TableOfContent
+    │   │   │   │   ├── common-components.jsx
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   └── ToastNotificationComponent
+    │   │   │       └── index.jsx
+    │   │   └── containers
+    │   │       ├── Dashboard
+    │   │       │   └── index.jsx
+    │   │       └── MainLayout
+    │   │           ├── index.jsx
+    │   │           └── index.less
+    │   ├── reducers
+    │   │   ├── authReducer.js
+    │   │   ├── datasetListReducer.js
+    │   │   ├── endpointReducer.js
+    │   │   ├── index.js
+    │   │   ├── loadingReducer.js
+    │   │   ├── navbarReducer.js
+    │   │   ├── overviewReducer.js
+    │   │   ├── tableOfContentReducer.js
+    │   │   ├── toastReducer.js
+    │   │   └── userProfileReducer.js
+    │   ├── store
+    │   │   └── store.js
+    │   ├── utils
+    │   │   ├── axiosInstance.js
+    │   │   ├── dateFunctions.js
+    │   │   ├── filterDataset.js
+    │   │   ├── helper.js
+    │   │   └── routeConstants.js
+    │   ├── App.css
+    │   ├── App.js
+    │   ├── App.test.js
+    │   ├── index.css
+    │   ├── index.js
+    │   ├── logo.svg
+    │   ├── reportWebVitals.js
+    │   └── setupTests.js
+    ├── config-overrides.js
+    ├── jsconfig.json
+    ├── jsdoc.json
+    ├── package.json
+    ├── package-lock.json
+    └── README.md
+```
+
+## Assets
+
+Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
+
+## Prerequisites
+
+## Install Node JS
+Refer to https://nodejs.org/en/ to install nodejs
```

Comment:
Rather than include the URL in the text, it would be friendlier to provide it as a hyperlink.  Also, you should probably omit the `/en/` suffix -- the top level seems to redirect, and I assume it does so more intelligently than to assume that the reader wants English.

Reply:
Done